<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="AccountLogin">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Вхід</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Registration">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Реєстрація</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="Ok">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cancel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Скасувати</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>Логін:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ShowPassword">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>121</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Показати пароль&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/images/Eye.png) 0 0 0 0 stretch stretch;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>Ok</zorder>
     <zorder>Cancel</zorder>
     <zorder>LoginLine</zorder>
     <zorder>PasswordLine</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>ShowPassword</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;Зареєструватися як&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="JobSeeker">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Здобувач</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Employer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Роботодавець</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>AccountLogin</tabstop>
  <tabstop>Registration</tabstop>
  <tabstop>LoginLine</tabstop>
  <tabstop>PasswordLine</tabstop>
  <tabstop>Ok</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>JobSeeker</tabstop>
  <tabstop>Employer</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
